### PR TITLE
Docs v2

### PR DESCRIPTION
**Pull request by adidahiya**

### CODE REVIEW

**Comment by adidahiya** [comment]
After I merge this, I'll add a banner to the v1 site on release/1.x

**Comment by adidahiya** [comment]
image

version number will be fixed when I tag 2.0.0-beta.1

File: packages/docs-app/src/styles/_banner.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  background: $green3;
+  height: $banner-height;
+  text-align: center;
+  line-height: $banner-height;
+  color: $white;
+
+  &:hover {
+    background-color: $green2;
+    text-decoration: none;
+    color: $white;
```
Comment (cmslewis) [inline packages/docs-app/src/styles/_banner.scss]: Do we need to redeclare `color: $white` on hover?

File: packages/docs-app/src/styles/_banner.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  padding-top: $pt-navbar-height + $banner-height;
+}
+
+.docs-navbar {
+  top: $banner-height;
+}
+
+.docs-nav {
+  top: $pt-navbar-height + $banner-height;
+}
```
Comment (cmslewis) [inline packages/docs-app/src/styles/_banner.scss]: Missing newline?

File: packages/docs-app/src/styles/_banner.scss (hunk 1 above)
Comment (adidahiya) [reply packages/docs-app/src/styles/_banner.scss]: not sure, I just pasted this from the internal site

File: packages/docs-app/src/styles/_banner.scss (hunk 2 above)
Comment (adidahiya) [reply packages/docs-app/src/styles/_banner.scss]: stylelint handles this

**Pull request merged**